### ISSUE TITLE
Backspace Button for Delete Operations

### ISSUE THREAD

**Issue by erayzesen** [BoostIO/Boostnote#3664]
Hi all. 

I'm using macos keyboard and the cmd+del shortcut is impossible for me. 

Can you add another shortcut to this for us? Just like cmd+backspace or only backspace...etc

**Comment by Flexo013** [comment]
I have tested this on both Windows and Linux with 0.16.1. On those the functionality to remove an entire word with these keybindings works fine.

@erayzesen What version of Boostnote Legacy and MacOS are you using?

**Comment by erayzesen** [comment]
Hi, I'm using Macos Catalina. The version is v0.16.0

The problem is that, doesn't exist "delete" button in the macos [keyboard](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRpTo_MWNmTlvopZ1naA4UPm_y5cgF2N3M-gQ&usqp=CAU). So may you add  an alternative as "backspace" button for this ? 

Thank you.

**Comment by Flexo013** [comment]
What functionality would you want backspace to trigger?

Also, from what I can find online you can use fn+backspace to press delete with that keyboard.

**Comment by erayzesen** [comment]
Yes, I know, I can also press remove button in the GUI but convenience is not the goal here?. fn+backspace isn't an easy way to do this and most of softwares uses backspace to "delete" function in the macos. (Built-in macos softwares too).

For example, you may delete any note folders, notes with just the backspace button in Apple Notes. You can be sure what I am requesting is one of the macOS usage routines.

**Comment by Flexo013** [comment]
> What functionality would you want backspace to trigger?

Could you answer this?

Also if your goal is convenience then I would recommend using a keyboard with commonly found keys.

For now I will check with some other mac users if this is a common problem within software, to if this is worth changing.

**Comment by erayzesen** [comment]
>>What functionality would you want backspace to trigger?
Delete operations. 
>>Also if your goal is convenience then I would recommend using a keyboard with commonly found keys.
Thank you. But I have no problems with the keyboard in other software anyway. Usually, in software targeted by Macos, delete functions are assigned to the backspace key for macOS keyboards. I voiced this request just because I realized it was not in this software. 

It may be good for the software to be more compatible on the macOS side.

**Comment by Flexo013** [comment]
> > What functionality would you want backspace to trigger?
> 
> Delete operations.

Can you be more specific? Mention the specific shortcuts, mention the specific results of pressing those shortcuts. Are these delete operation already accessed by pressing the delete key, and so on?

**Comment by erayzesen** [comment]
Ok. Cmd+del shortcut is imposible for macos users. It would be nice to use backspace for all functions with this shortcut.

As you said, fn + backspace can be useful when the delete key is needed. Because I have to do that it is out of this shortcut now;
cmd + fn + backspace

In other words, it is easier to press the button in the gui instead of using this shortcut while deleting many notes and categories.

I recommend using the backspace button as the usual delete shortcut in almost all software in macos.

If this doesn't make sense (because in other systems this shortcut is defined as ctrl + delete), you can also provide an option as cmd + backspace. But this is also used as a file deletion shortcut in macOS explorer. So the healthiest should only be backspace.

But it would be more useful, suppose you sit down and delete dozens of notes and edit them. Something you can do with the backspace key in other software;
It doesn't seem very user-friendly to do with cmd + fn + backspace.
